src/makernote_int.hpp

```diff
@@ -622,7 +622,7 @@ TiffComponent* newCasio2Mn2(uint16_t tag, IfdId group, IfdId mnGroup);
   @param pRoot Pointer to the root component of the TIFF tree.
   @return An index into the array set, -1 if no match was found.
  */
-int sonyCsSelector(uint16_t tag, const byte* pData, size_t size, TiffComponent* const pRoot);
```

Comment:
Same comment in all these functions insider `makernote_int`. I feel unconfortable having different signatures in the declaration and definitions. It might add unnecessary complexity for the code reader without much benefit.

Reply:
Done.